### PR TITLE
Update client version 3.13.24-dremio-0

### PR DESCRIPTION
**Pull request by omida-bitquill**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <groupId>net.snowflake</groupId>
   <artifactId>snowflake-jdbc</artifactId>
-  <version>3.13.24</version>
+  <version>3.13.24-dremio</version>
```
Comment (jlarue26) [inline pom.xml:7]:
```suggestion
  <version>3.13.24-dremio-0</version>
```

File: src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public class SnowflakeDriver implements Driver {
   static SnowflakeDriver INSTANCE;
 
   public static final Properties EMPTY_PROPERTIES = new Properties();
-  public static String implementVersion = "3.13.24";
+  public static String implementVersion = "3.13.24-dremio-0";
```
Comment (jduo) [inline src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java:26]: Should this come from a properties file? Should it include the git commit SHA1? I have concerns about how this should be formatted as we make more change.

Comment (jlarue26) [reply src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java:26]: We will create a GitHub issue to address this after 23.1.

Comment (jlarue26) [reply src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java:26]: We will go with snowflake-jdbc-3.13.24-dremio-0.jar

**Review by normanj-bitquill — APPROVED** [review]

**Pull request merged**